### DISCUSSION TITLE
Accessing Character Count of Output w/o Markup/HTML Tags

### DISCUSSION THREAD

**Discussion by sergebenard** [markedjs/marked#2331] [Q&A]
Hello!

I _love_ this library.

What I'm looking for is a way to count the characters of the output, but omit from the count HTML tags, as well as any Markup syntax.

_Example:_
```javascript
let text ='_Hello_ **World** !';
console.debug(text.length); // 17

marked(text) // <em>Hello</em> <strong>World</strong>!

console.debug(marked(text).length); // 45
```

I have found solutions which could work, but I'd rather use native functionality if it exists.

Is there a native function to help with what I need?

Thank you!

Serge

**Comment by UziTech** [comment]
You could try to use a [text renderer](https://github.com/markedjs/marked/issues/2320) then get the length of the output.

**Reply by sergebenard** [reply]
Oh there are a thousand different ways to make it work, but I’d like to use native functionality if it’s available.

Thanks for the reply!
